builder_test.go

```diff
@@ -633,6 +802,52 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 					})
 				})
 			})
+
+			when("invalid unmet entries", func() {
+				it.Before(func() {
```

Comment:
Should this also have a `when("platform API >= 0.5", ...`?

Reply:
Probably, just to be consistent. In fact we should probably change the implementation so that we actually aren't validating `build.toml` when it is written by an older buildpack.

Reply:
Sounds good. I should have said `buildpack API` above but same idea :)